runtime/test/zserio/json_test/json_test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "base.hpp"
+
+#include <zsr/find.hpp>
+#include <zsr/types-json.hpp>
+#include <zsr/introspectable-json.hpp>
+
+#include <nlohmann/json.hpp>
+
+namespace nlo = nlohmann;
+
+namespace {
+
+PKG;
+
+TEST(JsonTest, serialize_meta_subtype)
+{
+    auto s1 = zsr::find<zsr::SubType>(pkg, "S1");
+    auto s1j = nlo::json(*s1);
+
+    ASSERT_EQ(s1j["ident"], "S1");
+    ASSERT_EQ(s1j["type"]["ztype"]["type"], "int");
+}
+
+/* TODO: Add more useful tests */
```

Comment:
Maybe describe how the current test is lacking and what a "more useful" test should do 🤔